resources/js/Pages/Welcome.tsx

```diff
@@ -12,7 +12,7 @@ export default function Welcome({ auth }: PageProps) {
                         <Brand />
                     </div>
                     <div className="flex-none">
-                        <ul className="menu menu-horizontal px-1 text-white">
+                        <ul className="menu menu-horizontal px-1 text-teal-500">
```

Comment:
Great catch. There are actually theme specific values that would probably be better to use (found in the tailwind.config.js file.) For this I would probably going with `text-primary` instead of `text-teal-500`.